### PR TITLE
DOCS-SE-CC-3.17

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CC 3.17

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+**Event details page**
+
+Provides actions to remediate the detection and stop the attack from progressing. For example:
+
+![runtime-security](/img/calico-cloud/runtime-security-events.png)
+
+**Severity**
+
+{{prodname}} calculates severity (Critical, High, Medium, Low) using a combination of NIST CVSS 3.0 and MITRE IDs.
```
Comment (gantony) [inline calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx]: Not sure how accurate that statement is. We (security researcher) provide a severity score that we then map a level inspired by the thresholds of NIST CVSS 3.0. I don’t think Mitre IDs have a direct input in the severity score.

When I read it I get the impression that Mitre IDs contain some kind of severity information. My understanding is that it’s not the case and this is a judgement call from our security researcher. @anthonytwh do you have any thoughts on that?

File: calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+**MITRE IDs**
```
Comment (gantony) [inline calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx]: I think this category probably makes more sense below mitre tactics as that’s how they’re organised in the mitre framework…

File: calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx (hunk 1 above)
Comment (anthonytwh) [reply calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx]: Yeah MITRE Tactic or CVSS are not used for the severity of a detector. It is an arbitrary number we assign at the moment (no caluclations). 

I think we can simplify this wording somewhat along the lines of: a severity of an alert is a score given to a detection based upon the risk assessment of the alert.

File: calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx (hunk 2 above)
Comment (anthonytwh) [reply calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx]: There are MITRE IDs for Tactics and Techniques, perhaps these should be specified as such.

File: calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx]: I changed this to: 

Alert severity is a score given to a detection (Critical, High, Medium, Low), based upon the risk assessment of the alert.

File: calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/threat/security-event-management.mdx]: I shifted text to follow screenshot order.

**Review by gantony — APPROVED** [review]
LGTM! Thanks Barbara

**Pull request merged**